packages/zoe/src/typeGuards.js

```diff
@@ -144,17 +144,6 @@ export const ZoeMintI = M.interface('ZoeMint', {
   withdrawAndBurn: M.call(AmountShape).returns(),
 });
 
-export const ZcfMintI = M.interface('ZcfMint', {
```

Comment:
consider maintaining backwards compatibility.
```
/** @deprecated */
export { ZcfMintI } from 'contractFacet/typeGuards.js';
```

Reply:
Tried, but was immediately and correctly warned that it would introduce a cyclic dependency since `./contractFacet/typeGuards.js` sensibly imports from this file.

Since there are no static errors in this PR, there cannot be any imports broken by this absence in the agoric-sdk repo. Since the exports from this `./typeGuards.js` are not reexported by a de-jure exports, it cannot break any de-jure imports outside the agoric-sdk repo, although it might break deep imports. My inclination is that this is unlikely enough to not worry about it. Resolving on that basis, but if you disagree please unresolve.